I, [2018-10-17T08:09:40.900335 #19834]  INFO -- default-ubuntu-1604: -----> Creating <default-ubuntu-1604>...
I, [2018-10-17T08:09:51.511249 #19834]  INFO -- default-ubuntu-1604: The provider 'virtualbox' that was requested to back the machine
I, [2018-10-17T08:09:51.545620 #19834]  INFO -- default-ubuntu-1604: 'default' is reporting that it isn't usable on this system. The
I, [2018-10-17T08:09:51.545974 #19834]  INFO -- default-ubuntu-1604: reason is shown below:
I, [2018-10-17T08:09:51.546025 #19834]  INFO -- default-ubuntu-1604: 
I, [2018-10-17T08:09:51.546068 #19834]  INFO -- default-ubuntu-1604: Vagrant could not detect VirtualBox! Make sure VirtualBox is properly installed.
I, [2018-10-17T08:09:51.546117 #19834]  INFO -- default-ubuntu-1604: Vagrant uses the `VBoxManage` binary that ships with VirtualBox, and requires
I, [2018-10-17T08:09:51.546183 #19834]  INFO -- default-ubuntu-1604: this to be available on the PATH. If VirtualBox is installed, please find the
I, [2018-10-17T08:09:51.546225 #19834]  INFO -- default-ubuntu-1604: `VBoxManage` binary and add it to the PATH environmental variable.
E, [2018-10-17T08:09:51.681329 #19834] ERROR -- default-ubuntu-1604: Create failed on instance <default-ubuntu-1604>.
E, [2018-10-17T08:09:51.681643 #19834] ERROR -- default-ubuntu-1604: ------Exception-------
E, [2018-10-17T08:09:51.681782 #19834] ERROR -- default-ubuntu-1604: Class: Kitchen::ShellOut::ShellCommandFailed
E, [2018-10-17T08:09:51.681873 #19834] ERROR -- default-ubuntu-1604: Message: Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: 
STDERR: The provider 'virtualbox' that was requested to back the machine
'default' is reporting that it isn't usable on this system. The
reason is shown below:

Vagrant could not detect VirtualBox! Make sure VirtualBox is properly installed.
Vagrant uses the `VBoxManage` binary that ships with VirtualBox, and requires
this to be available on the PATH. If VirtualBox is installed, please find the
`VBoxManage` binary and add it to the PATH environmental variable.
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1
E, [2018-10-17T08:09:51.681928 #19834] ERROR -- default-ubuntu-1604: ----------------------
E, [2018-10-17T08:09:51.682327 #19834] ERROR -- default-ubuntu-1604: ------Backtrace-------
E, [2018-10-17T08:09:51.682377 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/shell_out.rb:71:in `rescue in run_command'
E, [2018-10-17T08:09:51.682427 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/shell_out.rb:58:in `run_command'
E, [2018-10-17T08:09:51.682476 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:452:in `run_command'
E, [2018-10-17T08:09:51.682525 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:399:in `run'
E, [2018-10-17T08:09:51.682574 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:483:in `run_vagrant_up'
E, [2018-10-17T08:09:51.682624 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:109:in `create'
E, [2018-10-17T08:09:51.682739 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:484:in `public_send'
E, [2018-10-17T08:09:51.682794 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:484:in `block in perform_action'
E, [2018-10-17T08:09:51.682844 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:548:in `block in synchronize_or_call'
E, [2018-10-17T08:09:51.682910 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:546:in `synchronize'
E, [2018-10-17T08:09:51.682961 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:546:in `synchronize_or_call'
E, [2018-10-17T08:09:51.683011 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:513:in `block in action'
E, [2018-10-17T08:09:51.683060 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/2.5.0/benchmark.rb:293:in `measure'
E, [2018-10-17T08:09:51.683108 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:512:in `action'
E, [2018-10-17T08:09:51.683183 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:484:in `perform_action'
E, [2018-10-17T08:09:51.683238 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:394:in `create_action'
E, [2018-10-17T08:09:51.683286 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:382:in `block (2 levels) in transition_to'
E, [2018-10-17T08:09:51.683335 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/lifecycle_hooks.rb:45:in `run_with_hooks'
E, [2018-10-17T08:09:51.683383 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:381:in `block in transition_to'
E, [2018-10-17T08:09:51.683432 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:380:in `each'
E, [2018-10-17T08:09:51.683479 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:380:in `transition_to'
E, [2018-10-17T08:09:51.683529 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:129:in `create'
E, [2018-10-17T08:09:51.683578 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/command.rb:197:in `public_send'
E, [2018-10-17T08:09:51.683626 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2018-10-17T08:09:51.683739 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2018-10-17T08:09:51.683793 #19834] ERROR -- default-ubuntu-1604: ----End Backtrace-----
E, [2018-10-17T08:09:51.683842 #19834] ERROR -- default-ubuntu-1604: ---Nested Exception---
E, [2018-10-17T08:09:51.683889 #19834] ERROR -- default-ubuntu-1604: Class: Mixlib::ShellOut::ShellCommandFailed
E, [2018-10-17T08:09:51.683936 #19834] ERROR -- default-ubuntu-1604: Message: Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: 
STDERR: The provider 'virtualbox' that was requested to back the machine
'default' is reporting that it isn't usable on this system. The
reason is shown below:

Vagrant could not detect VirtualBox! Make sure VirtualBox is properly installed.
Vagrant uses the `VBoxManage` binary that ships with VirtualBox, and requires
this to be available on the PATH. If VirtualBox is installed, please find the
`VBoxManage` binary and add it to the PATH environmental variable.
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1
E, [2018-10-17T08:09:51.683991 #19834] ERROR -- default-ubuntu-1604: ----------------------
E, [2018-10-17T08:09:51.684050 #19834] ERROR -- default-ubuntu-1604: ------Backtrace-------
E, [2018-10-17T08:09:51.684099 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/shell_out.rb:71:in `rescue in run_command'
E, [2018-10-17T08:09:51.684147 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/shell_out.rb:58:in `run_command'
E, [2018-10-17T08:09:51.684232 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:452:in `run_command'
E, [2018-10-17T08:09:51.684287 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:399:in `run'
E, [2018-10-17T08:09:51.684338 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:483:in `run_vagrant_up'
E, [2018-10-17T08:09:51.684386 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/kitchen-vagrant-1.3.4/lib/kitchen/driver/vagrant.rb:109:in `create'
E, [2018-10-17T08:09:51.684434 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:484:in `public_send'
E, [2018-10-17T08:09:51.684482 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:484:in `block in perform_action'
E, [2018-10-17T08:09:51.684529 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:548:in `block in synchronize_or_call'
E, [2018-10-17T08:09:51.684577 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:546:in `synchronize'
E, [2018-10-17T08:09:51.684625 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:546:in `synchronize_or_call'
E, [2018-10-17T08:09:51.684743 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:513:in `block in action'
E, [2018-10-17T08:09:51.684798 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/2.5.0/benchmark.rb:293:in `measure'
E, [2018-10-17T08:09:51.684854 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:512:in `action'
E, [2018-10-17T08:09:51.684904 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:484:in `perform_action'
E, [2018-10-17T08:09:51.684953 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:394:in `create_action'
E, [2018-10-17T08:09:51.685001 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:382:in `block (2 levels) in transition_to'
E, [2018-10-17T08:09:51.685049 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/lifecycle_hooks.rb:45:in `run_with_hooks'
E, [2018-10-17T08:09:51.685097 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:381:in `block in transition_to'
E, [2018-10-17T08:09:51.685145 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:380:in `each'
E, [2018-10-17T08:09:51.685223 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:380:in `transition_to'
E, [2018-10-17T08:09:51.685273 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/instance.rb:129:in `create'
E, [2018-10-17T08:09:51.685331 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/command.rb:197:in `public_send'
E, [2018-10-17T08:09:51.685382 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/command.rb:197:in `run_action_in_thread'
E, [2018-10-17T08:09:51.685429 #19834] ERROR -- default-ubuntu-1604: /opt/chefdk/embedded/lib/ruby/gems/2.5.0/gems/test-kitchen-1.23.2/lib/kitchen/command.rb:169:in `block (2 levels) in run_action'
E, [2018-10-17T08:09:51.685477 #19834] ERROR -- default-ubuntu-1604: ----End Backtrace-----
